synthtool/gcp/common.py

```diff
@@ -250,4 +248,4 @@ def _load_repo_metadata(metadata_file: str = "./.repo-metadata.json") -> Optiona
     if os.path.exists(metadata_file):
         with open(metadata_file) as f:
             return json.load(f)
-    return None
+    return {}
```

Comment:
If this only ever returns a dictionary, we should change the type hint from `Optional[Dict]` -> `Dict`.